Vungle/VungleBannerCustomEvent.m

```diff
@@ -68,6 +68,22 @@ - (void) invalidate
     [[VungleRouter sharedRouter] invalidateBannerAdViewForPlacementID:self.placementId delegate:self];
 }
 
+- (CGSize)sizeForCustomEventInfo:(CGSize)size
+{
+    CGFloat width = size.width;
+    CGFloat height = size.height;
+
+    if (height >= kVNGLeaderboardBannerSize.height && width >= kVNGLeaderboardBannerSize.width) {
+        return CGSizeMake(kVNGLeaderboardBannerSize.width, kVNGLeaderboardBannerSize.height);
```

Comment:
With each return in this condition if section - why not directly return the size object that has been defined in VungleRouter? Calling CGSizeMake using width and height from an existing constant CGSize seems to be adding an unnecessary step.

Reply:
@crbinsf Hi Clarke, thanks for the good advice. I will update it right now. ^_^

Reply:
@crbinsf Done, thanks. 🙂